github/orgs_teams.go

```diff
@@ -136,11 +136,6 @@ type NewTeam struct {
 	//     closed - visible to all members of this organization
 	// Default is "secret".
 	Privacy *string `json:"privacy,omitempty"`
-
-	// TODO: decide whether to delete
-	// LDAPDN is only available in GitHub Enterprise and when the team
-	// membership is synchronized with LDAP.
-	LDAPDN *string `json:"ldap_dn,omitempty"`
```

Comment:
@e-beach I don't think `LDAPDN` should be deleted. On GitHub Enterprise, it's allowed to add or update a team with this parameter. We use it in the Terraform GitHub provider too.

Reply:
Thanks for that feedback! I'll add it back in today, whoops.

Reply:
Thanks @e-beach!